database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -117,7 +121,7 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
       })
     } yield counts.sum
     // Auto-commit mode, so each statement is individually committed to avoid deadlocks
-    runAction(action)
+    runTransaction(action)
```

Comment:
per the comment the non-transactionality is kinda important 🙂

Reply:
This is a hard requirement of Postgres due to `importsZip: Option[Blob]` in `WorkflowStoreEntry`.  I can change it to only run transactionally for Postgres if you'd prefer, but it definitely won't work any other way.

Reply:
Actually I'd better double-check this as well.  I may have gotten carried away converting calls to run transactionally - although there are definitely at least one or two cases where it's essential.

Reply:
This one does appear to be unnecessary, so I pushed a revert.